src/games/_gameswidget.py

```diff
@@ -133,6 +133,10 @@ def __init__(self, client, *args, **kwargs):
 
         self.gameList.setItemDelegate(GameItemDelegate(self))
         self.gameList.itemDoubleClicked.connect(self.gameDoubleClicked)
+        self.gameList.sortBy = 0 # Default Sorting is By Players count
+
+        self.sortGamesComboBox.addItems(['By Players', 'By Game Quality', 'By avg. Player Score'])

```

Comment:
Minor detail here is that option 3 should be "By avg. Player Rating", since that appears to be what the option does.